phpBB/phpbb/cache/driver/void.php

```diff
@@ -16,7 +16,7 @@
 /**
 * ACM Null Caching
 */
-class null extends \phpbb\cache\driver\base
+class void extends \phpbb\cache\driver\base
```

Comment:
I think we should keep the old null class (extending the new one) for BC purpose